doc/rst/language/spec/procedures.rst

```diff
@@ -530,8 +529,8 @@ intents:
 ================================ ====== ========= ========= =========== ============ =============
 \                                ``in`` ``out``   ``inout`` ``ref``     ``const in`` ``const ref``
 ================================ ====== ========= ========= =========== ============ =============
-copied in on function call?      yes    no        yes       no          yes          no
-copied out on function return?   no     yes       yes       no          no           no
+can copy in on function call?    yes    no        yes       no          yes          no
+can copy out on function return? no     yes       yes       no          no           no
```

Comment:
This makes me think "so, it is not ALWAYS copied in/out?" You are probably hinting at transferring ownership instead of copying when possible. Even if so, I prefer the existing wording.

Also, I suggest using "assign" instead of "copy" for function returns, if this is coherent with the definition of the out-intent. Perhaps "assigned to actual upon return", dropping "function" for brevity.

Reply:
> This makes me think "so, it is not ALWAYS copied in/out?"

No, it is not, because we have copy elision. We can say "formal is initialized from actual" without saying copy-initialized or move-initialized, so I will do that.

`out` is not always assigned since in some cases the actual can be move-initialized from the out formal according to split initialization.